src/vs/workbench/contrib/chat/browser/actions/chatCodeblockActions.ts

```diff
@@ -370,7 +370,7 @@ export function registerChatCodeBlockActions() {
 			const editorService = accessor.get(IEditorService);
 			const chatService = accessor.get(IChatService);
 
-			editorService.openEditor(<IUntitledTextResourceEditorInput>{ contents: context.code, languageId: context.languageId, resource: undefined });
+			editorService.openEditor({ contents: context.code, languageId: context.languageId, resource: undefined } satisfies IUntitledTextResourceEditorInput);
```

Comment:
Switch to use `satisfies` here to check the type without actually changing its shape